app/views/catalog/_home.html.erb

```diff
@@ -1,29 +1,88 @@
+        <p>Images from Princeton's map collection. Includes historic, geoscience, topographic, and bathymetric maps and charts as well as maps in political, physical, and cultural geography.</p>
+      </a>
+      <a href="/?f%5Bdc_format_s%5D%5B%5D=Scanned+Map&q=Sanborn" class="grid-item">
+        <span class="icon-building"></span>
+        <h1>Sanborn</h1>
+        <p>Detailed fire insurance maps of cities and towns in New Jersey. Created by The Sanborn Map Company, the oldest mapping company in the United States.</p>
+      </a>
+      <a href="/?f%5Bdc_format_s%5D%5B%5D=Shapefile&q=parcels" class="grid-item">
+        <span class="icon-locationmap"></span>
+        <h1>Parcels</h1>
+        <p>Property boundry data for identification of ownership, land use, and land value.</p>
+      </a>
+      <a href="/?f%5Bdc_subject_sm%5D%5B%5D=population" class="grid-item">
+        <span class="icon-graphbar"></span>
+        <h1>Census</h1>
+        <p>Worldwide population and demographic data.</p>
+      </a>
+        <a href="/?f%5Bdct_spatial_sm%5D%5B%5D=Princeton&q=" class="grid-item">
+        <span class="geoblacklight-icon geoblacklight-princeton"></span>
+        <h1>Princeton</h1>
+        <p>Data and maps related Princeton University.</p>
+      </a>
     </div>
-    <div class="bullet-content">
-      <h2>Geographic Data</h2>
-      <p>Base and thematic spatial data from around the world. Topics in human geography (political boundaries, transportation, population) as well as and physical geography (geologic, elevation).</p>
-      </div>
-  </li>
-  <li class="bullet" onclick="location.href='catalog?f%5Bdc_format_s%5D%5B%5D=Scanned+Map&q=-Sanborn';">
-    <div class="bullet-icon bullet-icon-2">
-      <img src="https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/placeholder_logo_4.png" alt="">
-    </div>
-    <div class="bullet-content">
-      <h2>Scanned Maps</h2>
-      <p>Images from Princeton's map collection. Includes historic, geoscience, topographic, and bathymetric maps and charts as well as maps in political, physical, and cultural geography.</p>
-    </div>
-  </li>
-  <li class="bullet" onclick="location.href='catalog/?f%5Bdc_format_s%5D%5B%5D=Scanned+Map&q=Sanborn';">
-    <div class="bullet-icon bullet-icon-3">
-      <img src="https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/placeholder_logo_3.png" alt="">
+  </div>
+</section>
+
+<section class="topics">
+  <div class="container">
+    <div class="line-behind-text">
+      <h6 class="er_toc_tag" id="er-toc-id-2">Featured Maps</h6>
     </div>
-    <div class="bullet-content">
-      <h2>Sanborn</h2>
-      <p>Detailed building maps of cities and towns in New Jersey. Created by The Sanborn Map Company, the oldest mapping company in the United States, primarily for use by insurance companies.</p>
+
+    <div class="cards">
+      <div class="card">
+        <a class="nostyle" href="/catalog/princeton-2v23vw81v">
```

Comment:
Yes, those came out nice. I was thinking that we could have Axa look over the app and make some notes about how she would improve the ui. I did my best.